tests/schema_utils.py

```diff
@@ -0,0 +1,22 @@
+import sys
+from pathlib import Path
+
+
+class SchemasDirectoryNotFound(Exception):
+    pass
+
+
+def get_schemas_base_path(lib_name: str = 'libero-schemas') -> str:
```

Comment:
maybe this module should be placed into `src/` as it could be useful also on the application side rather than the tests? Could envisage a debug/testing mode where generated responses would be validated, but also where if requests are made to another API those responses can be validated too.

Reply:
Schemas are only used in this project to test that generate XML is valid. Therefore, I consider it a testing code. If I need to use these utils in the sections of code that are executed at run time then I was reconsider the modules location. I think eventually this will be moved out into a library if I understand @thewilkybarkid plan.

Reply:
Yes, can be reevaluated in the future if it becomes of more general use